pyproject.toml

```diff
@@ -8,6 +8,7 @@ license = "MIT"
 [tool.poetry.dependencies]
 python = "^3.8,<3.11"
 defusedxml = "^0.7.1"
+humanize = "^4.7.0"
```

Comment:
Why is `humanize` here as well?
Isn't it enough that's in `tool.poetry.group.ci.dependencies`?